From 00284034394305135bee71de3267b82fc5ad0c83 Mon Sep 17 00:00:00 2001
From: b02247 <b02247@freescale.com>
Date: Thu, 10 Jan 2013 14:30:38 +0800
Subject: [PATCH 144/602] ENGR00235540  remove gps setting in init.rc

The gpio60 is used by the touchsreen. gps set it in init.rc which will
cause the touchsreen can not be used.

Signed-off-by: b02247 <b02247@freescale.com>
---
 sabreauto_6q/init.rc | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/sabreauto_6q/init.rc b/sabreauto_6q/init.rc
index f561c37..b0a854f 100644
--- a/sabreauto_6q/init.rc
+++ b/sabreauto_6q/init.rc
@@ -22,14 +22,6 @@ on boot
     # Set permission for IIM node
     symlink /dev/mxs_viim /dev/mxc_mem
 
-    # Set GPS serial and reset GPIO pin
-    setprop ro.kernel.android.gps /dev/athrnmea
-    setprop persist.gps.oacmode @f
-    symlink /dev/ttymxc2 /dev/gpsdevice
-    write /sys/class/gpio/export 60
-    write /sys/class/gpio/gpio60/direction "out"
-    write /sys/class/gpio/gpio60/value 1
-
     # Enable Tethering in the Settings
     setprop ro.tether.denied false
 
-- 
1.8.0

